🤫 Secrets/Config Vars (#636)
cloudflare/workers-docs: content/tooling/wrangler/configuration/environments.md

- [Concepts](#concepts)
- [Usage](#usage)
  - [Examples](#examples)
  - [Custom webpack configurations](#custom-webpack-configurations)

>   - [Environment Variables](#environment-variables)

Reviewer: I'd just call this `Environments` to avoid confusion between this and text vars.

Your default `wrangler build`, `wrangler preview`, and `wrangler publish` commands will all build with `webpack.dev.js`, as will `wrangler build -e staging`, `wrangler preview -e staging`, and `wrangler publish -e staging`. `wrangler build -e production`, `wrangler preview -e production`, and `wrangler publish -e production` would all use your `webpack.config.js` file.

### Configuration Variables

> You can specify different KV namespaces and text variables for different environments.

Reviewer 2: ```suggestion
You can specify different KV namespaces and configuration variables for different environments.
```

kv-namespaces = [
    { binding = "KV", id = "bd46d6484b665e6bd134b0496ad97760" }
]

> vars = {

Reviewer: I'd put an example `vars` entry in the default option above (right under the kv-namespaces from line 226-228).

> config_vars = {

Reviewer 2: ```suggestion
config = {
```

  FOO = "some text",
}
```

> Note: Secret variables are assigned to specific environments by default when using the [`wrangler secret create`](/tooling/wrangler/secret/) command.

Reviewer: ```suggestion
Note: Secret variables can be assigned to specific environments by passing the `-e/--env <environment_name>` flag while using the [`wrangler secret create`](/tooling/wrangler/secret/) command.
```

> Note: Secret variables are assigned to specific environments by default when using the [`wrangler secrete create`](/tooling/wrangler/secret/) command.

Reviewer 2: ```suggestion
Note: Secret variables are assigned to specific environments by default when using the [`wrangler secret create`](/tooling/wrangler/secret/) command.
```

Reviewer 2: unfortunate typo lol
